firmware/throttle/src/throttle_control.cpp

```diff
@@ -163,6 +132,7 @@ void disable_control( void )
     if( g_throttle_control_state.enabled == true )
     {
         const uint16_t num_samples = 20;
+        //TODO: Should this be inside the cli()?!?
```

Comment:
For disabling control I think that we might need to disable interrupts, but again this should only be needed in `oscc_dac.cpp`.

As I think about it more, though, I think that this operation itself might cause a voltage discontinuity. Let's say that OSCC is applying the brakes when a disable is triggered, and the brakes aren't manually depressed. This operation will sample the disconnected brakes which have a low value, apply that to the DAC which will immediately drop the voltage down to the pedal unpressed state (which is the definition of a voltage discontinuity), and then toggle the spoof relay.

So it looks like this operation is already broken. Neat, huh?

Reply:
Wow shouldn't have left the `TODO:` in there but really good points for further analysis of the DAC behavior. I'll remove the `TODO:` for now since we documented these comments internally.